superset/config.py

```diff
@@ -585,6 +588,59 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # dictionary.
 JINJA_CONTEXT_ADDONS: Dict[str, Callable] = {}
 
+# A dictionary of macro template processors that gets merged into global
+# jinja template processors. The existing template processors get updated with this
+# dictionary, which means the existing keys get overwritten by the content of this
+# dictionary. The customized addons don't necessarily need to use jinjia templating
+# language. This allows you to define custom logic to process macro template.
+#
+# The example below configures a custom presto template processor which implements
+# its own logic of processing macro template with regex parsing. It uses $ style
+# macro instead of Jinja's {{ }} style.
+# By configuring it with CUSTOM_TEMPLATE_PROCESSOR, sql template on presto database
+# is processed by the custom one rather than the default one.
+#
+# def DATE(
+#     ts: datetime, day_offset: SupportsInt = 0, hour_offset: SupportsInt = 0
+# ) -> str:
+#     """Current day as a string"""
+#     day_offset, hour_offset = int(day_offset), int(hour_offset)
+#     offset_day = (ts + timedelta(days=day_offset, hours=hour_offset)).date()
+#     return str(offset_day)
+#
+# class CustomPrestoTemplateProcessor(PrestoTemplateProcessor):
```

Comment:
Maybe move this example to the docs, and just leave a nice short description on the config

Reply:
+1 - This should be somewhere else -- could even just reference your test processor

Reply:
Ack.